protocol/cosi.go

```diff
@@ -76,7 +76,7 @@ type CommitmentHook func(in []abstract.Point) error
 type ChallengeHook func(ch abstract.Scalar) error
 
 // ResponseHook allows for handling what should happen when all
-// responses are received
+// responses are received and our response is calculated
```

Comment:
No, It's OK. I though it was about the callback, I mean handler ;-) and it's about the actual repsonse (which is indeed calculated)